Dockerfile

```diff
@@ -14,6 +14,11 @@ RUN bin/bash -l -c "gem install aws-sdk mime-types"
 RUN apt-get install -y python2.7
 ENV PYTHON /usr/bin/python2.7
 
+# install hugo
+ENV HUGO_VERSION 0.21
```

Comment:
@jeremyzilar I think another repo would be best because Federalist only allows selection of a single build engine for a repo, not on a per-branch basis.